apps/nextjs-website/src/components/Feature.tsx

```diff
@@ -1,8 +1,9 @@
 'use client';
 import { Feature as FeatureEC } from '@pagopa/pagopa-editorial-components';
 import { FeatureProps } from '@pagopa/pagopa-editorial-components/dist/components/Feature/Feature';
-import { Icon } from '@mui/material';
-import { formatValidMuiIcon, isValidMuiIcon } from './Icons';
+// Only temporarily importing every icon. A task is planned to sub this for a restricted set of accepted icon names.
+import * as MuiIcons from '@mui/icons-material';
+import { isValidMuiIcon } from './Icons';
```

Comment:
These imports shouldn't be required if you limit the possible values ​​for [icon](https://github.com/pagopa/b2b-portals/blob/d2d31b8b31a65afe3d42507b6c80384cee5519ae/apps/nextjs-website/src/lib/fetch/types/PageSection.ts#L72) as follows:

```typescript
      icon: t.union([
        t.literal('Instagram'),
        t.literal('LinkedIn'),
        t.literal('Facebook'),
        t.literal('Twitter'),
        t.null,
      ]),
```

Reply:
The icons allowed in the Feature section are not reduced to just social icons, that's only for the Footer's socialLinks.

We will reduce the icons allowed for the Feature Section Items with task [B2BP-348](https://pagopa.atlassian.net/jira/software/projects/B2BP/boards/482?selectedIssue=B2BP-348)